### PR TITLE
Utilities css

### PR DESCRIPTION
**Pull request by saketkum11**
Please review my utilities css do let  me know whether I  need to added somthing

### CODE REVIEW

**Review by ramalingaa — APPROVED** [review]
No changes required

File: utilities-css/spacing.css [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+    --margin-5: 1rem;
+    --margin-6: 1.25rem;
+    --margin-7: 1.5rem;
+    --margin-8: 1.75rem;
+    --margin-9: 2rem;
+    --margin-10: 2.25rem;
+    --margin-11: 2.5rem;
+
+
+
```
Comment (shraddha15-creator) [inline utilities-css/spacing.css:45]: Remove empty lines

File: utilities-css/spacing.css [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+
+
+
+
+
+
+
```
Comment (shraddha15-creator) [inline utilities-css/spacing.css]: Remove empty lines

**Review by shraddha15-creator — CHANGES_REQUESTED** [review]

File: utilities-css/spacing.css (hunk 1 above)
Comment (saketkum11) [reply utilities-css/spacing.css:45]: okkk

File: utilities-css/spacing.css (hunk 2 above)
Comment (saketkum11) [reply utilities-css/spacing.css]: sure will do that

**Review by shubhambishnoi — COMMENTED** [review]
good

**Pull request merged**